mono/tests/bug-60843.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+
+class A : Attribute
+{
+    public object X;
+
+    public static void Main()
+    {
+        var x = (C<int>.E)AttributeTest(typeof(C<>.E));
+        Assert(C<int>.E.V == x);
+        var y = (C<int>.E2[])AttributeTest(typeof(C<>.E2));
+        Assert(y.Length == 2);
```

Comment:
Maybe pass a `String` to each assert so we could tell from CI which piece of the test failed? or use exit codes like other `mono/tests` tests or the TestDriver.cs - style tests.